{{cookiecutter.project_slug}}/modules/urls.py

```diff
@@ -11,12 +12,13 @@
 
 try:
     modules_dir = f"{settings.BASE_DIR}/modules/"
-    urls = Path(modules_dir).rglob('urls.py')
+    urls = Path(modules_dir).rglob("urls.py")
     for url in urls:
-        module_name, _ = url.as_posix().split('/')[-2:]
+        module_name, _ = url.as_posix().split("/")[-2:]
         if not module_name == "modules":
+            module_url = module_name.replace("_", "-")
```

Comment:
The file system should have underscore too, not dashes. Why is this needed?

Reply:
@danielsousaio would be more suited to answer this https://github.com/crowdbotics/django-scaffold/pull/219/commits/5b1c16a8addf3bff0fb66c1c4d8bbd4ab4b16790